packages/common/tracing/__init__.py

```
@@ -0,0 +1,171 @@
+
+    def __enter__(self) -> str:
+        """Enter the context and set the correlation ID.
+
+        Returns:
+            str: The correlation ID for this context.
+        """
+        self.previous_id = get_correlation_id()
+        self.correlation_id = set_correlation_id(self.correlation_id)
+        return self.correlation_id
+
+    def __exit__(self, *args: Any) -> None:
```

Ruff (0.14.1)

Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)